src/javascripts/templates/template.js

```diff
@@ -0,0 +1,70 @@
+import { formatNumber } from "../helpers/helpers"
+
+class Template {
+    /**
+     * Constructor of Template object
+     */
+    constructor(){ }
+
+     /**
+     * HTML Template for render an Transaction object
+     * @param {object} transaction 
+     */
+
+    static transaction = (transaction) => `
+        <div class="transaction-detail">
+          <div class="transaction-info">
+            <div class="datetime">
+              <p class="date-text">${new Date(transaction.date).getDate()}</p>
+              <div>
+                <div class="daymonthyear">
+                  <p class="daymonthyear-text">${new Date(transaction.date).toLocaleDateString('en-US', { weekday: 'long' })},</p>
```

Comment:
Write helper or util for transforming data

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !